ptvsd/wrapper.py

```diff
@@ -2427,6 +2427,8 @@ def on_pydevd_thread_create(self, seq, args):
         # as well.
         with self.is_process_created_lock:
             if not self.is_process_created:
+                if not debugger_attached.isSet():
+                    return
```

Comment:
@int19h Need your input on this. Basically i was running into a race in tests where we would get `process` message before we even send `initialize` request. The process message came from this location. I think we should not send this event unless debugger attached.

Reply:
Agreed.